src/DataDog/AuditBundle/EventSubscriber/AuditSubscriber.php

```diff
@@ -70,9 +70,13 @@ public function getUnauditedEntities()
     {
         return array_keys($this->unauditedEntities);
     }
-
+    
     private function isEntityUnaudited($entity)
     {
+        if ($entity instanceof Association || $entity instanceof AuditLog) {
```

Comment:
Can't we reuse the `private $unauditedEntities = []` instead of checking implicitly?